libcalico-go/lib/backend/k8s/k8s_test.go

```diff
@@ -412,22 +412,34 @@ var _ = testutils.E2eDatastoreDescribe("Test Syncer API for Kubernetes backend",
 		syncer.Stop()
 	})
 
-	It("should handle a Namespace with DefaultDeny (v1beta annotation for namespace isolation)", func() {
+	It("should handle a Namespace with DefaultDeny (v1 annotation for namespace isolation)", func() {
```

Comment:
So, I think this test is actually obsolete at this point.

It was testing an annotation that was used in beta versions of the original network policy API that was eventually thrown out and shouldn't be used any more. 

That said, I think the original test was correct - it was verifying this annotation:

```
					"net.beta.kubernetes.io/network-policy": "{\"ingress\": {\"isolation\": \"DefaultDeny\"}}",
```

Which is no longer here. So we can probably just revert these test changes and then in a subsequent PR I'll look at removing this test

Reply:
Reverted.

Reply:
reverted